app/lib/frontend/templates/admin.dart

```diff
@@ -128,6 +165,11 @@ String renderAccountPublishersPage({
 Tab _myPackagesLink() =>
     Tab.withLink(id: 'packages', title: 'My packages', href: '/my-packages');
 
+Tab _myLikedPackagesLink() => Tab.withLink(
+    id: 'liked-packages',
+    title: 'My liked packages',
+    href: '/my-liked-packages');
```

Comment:
Please add a helper method for this in `shared/urls.dart`.

Reply:
Done.